### ISSUE TITLE
Listbox selection behavior results in unexpected consequences

### ISSUE THREAD

**Issue by shawwn** [Tarsnap/tarsnap-gui#211]
I decided to open an issue for the problem mentioned in #209:

> When you click on a listbox item A, and then click on another item B, the UI tries to select every item from A to B inclusive, rather than switching the selection from A to B. It's a bit confusing to try to select a single item, and unexpected since shift isn't held down.

My Backup tab looks like this:

![image](https://user-images.githubusercontent.com/59632/44621634-fb40d880-a86e-11e8-9cf3-a4ba28388dce.png)

I tried to create a couple archives of `disruption`, and the result was this:

![image](https://user-images.githubusercontent.com/59632/44621643-1c092e00-a86f-11e8-9722-836d38e79125.png)

Notice that the size is 157MB instead of 171KB.

If you'd asked me "What does the archive `disruption1` contain?" I would have told you "the contents of ~/disruption and nothing else". That was incorrect -- it contained both ~/bin and ~/disruption.

There are several ways to mitigate this:

- Ignore it (which is a valid engineering tradeoff)
- Ensure that the listbox has only two colors: *selected* and *unselected*. Currently it has three (Edit: four, actually). For each of these screenshots, consider the perspective of a new user trying to answer the question "Which items are selected?"

Screenshot A:
![image](https://user-images.githubusercontent.com/59632/44621723-e2d1bd80-a870-11e8-9ad6-1d330c94b8be.png)

Screenshot B:
![image](https://user-images.githubusercontent.com/59632/44621772-eade2d00-a871-11e8-9d43-d18142971477.png)

Screenshot C:
![image](https://user-images.githubusercontent.com/59632/44621728-f715ba80-a870-11e8-8375-0b61b94b1f26.png)

With no context, presented to a new user, I think users would say that in Screenshot A, the `bin` directory is selected. In Screenshot C, they might reason that the `bin` directory is selected and `disruption` is not, since if they were both selected, they would be the same color. And Screenshot B would have a clear answer -- "`disruption` is selected, but `bin` is not" -- except the UI at the bottom shows that the total size would be 154.28MB if they were to click "Backup" whereas the size of `disruption` is 171.71KB.

I think the fact that `154.28MB` never changes is just a bug. Yup, confirmed: regardless of my listbox selection, Tarsnap always creates an archive containing both `bin` and `disruption`. I'll open a new issue for that.

To sum up, there seems to be two things to address: presentation (colors) and interaction (clicks). I think it would be clearer if there were only two colors (selected and unselected), and that clicking without holding shift will always result in a single item being selected, not a range of items.

**Comment by shawwn** [comment]
I was confused. I see now that the listbox selection on the Backups tab is totally irrelevant. I thought the Backups tab was intended to be a list of all the folders you might care about, and then you can periodically create archives containing some subset of the list. Never mind!
